pkgs/by-name/jp/jpeg2png/package.nix

```diff
@@ -0,0 +1,52 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  libjpeg,
+  libpng,
+  nix-update-script,
+  versionCheckHook,
+  zlib,
+}:
+stdenv.mkDerivation rec {
+  pname = "jpeg2png";
+  version = "1.02";
+
+  src = fetchFromGitHub {
+    owner = "ThioJoe";
+    repo = "jpeg2png";
+    rev = "v${version}";
+    hash = "sha256-GTc2r9w87PUKQqhPPOD26smpqt6fIH4QyM14hmfYB1Q=";
+  };
+
+  buildInputs = [
+    libjpeg
+    libpng
+    zlib
+  ];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p $out/bin
+    mv jpeg2png $out/bin/
+    runHook postInstall
+  '';
+
+  doInstallCheck = true;
+  nativeInstallCheckInputs = [ versionCheckHook ];
+
+  passthru = {
+    updateScript = nix-update-script { };
+  };
+
+  meta = {
+    description = "Silky smooth JPEG decoding";
+    homepage = "https://github.com/ThioJoe/jpeg2png";
+    changelog = "https://github.com/ThioJoe/jpeg2png/releases/tag/v${version}";
+    license = lib.licenses.gpl3Plus;
+    platforms = with lib.platforms; unix;
```

Comment:
```suggestion
    platforms = lib.platforms.unix;
```

Reply:
Looks like Windows is actually supported too, so I'll add that here.